src/kOS/Module/kOSProcessor.cs

```diff
@@ -75,6 +75,19 @@ public string Tag
         [KSPField(isPersistant = true, guiActive = false)]
         public int MaxPartId = 100;
```

Comment:
This is *kind* of a magic number.  Perhaps defining constants should be done all together, so when someone balances this they don't have to hunt all over the file.